restcomm/restcomm.interpreter/src/main/java/org/restcomm/connect/interpreter/BaseVoiceInterpreter.java

```diff
@@ -1517,11 +1531,42 @@ public Gathering(final ActorRef source) {
 
         @Override
         public void execute(final Object message) throws Exception {
+
+            // check mandatory attribute partialResultCallback
+            Attribute resultCallbackAttr = verb.attribute(GatherAttributes.ATTRIBUTE_PARTIAL_RESULT_CALLBACK);
             final NotificationsDao notifications = storage.getNotificationsDao();
+            if (resultCallbackAttr == null) {
```

Comment:
@DmytroDev why when `ATTRIBUTE_PARTIAL_RESULT_CALLBACK` is null we stop Interpreter? I believe we should stop only when `ATTRIBUTE_ACTION` is null. There might be cases where application doesn't need to know about partial results but for the final result. Isn't that the case?